addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/SerialMessage.java

```diff
@@ -154,7 +158,13 @@ public SerialMessage(int nodeId, byte[] buffer) {
         }
         this.priority = SerialMessagePriority.High;
         this.messageType = buffer[2] == 0x00 ? SerialMessageType.Request : SerialMessageType.Response;
-        this.messageClass = SerialMessageClass.getMessageClass(buffer[3] & 0xFF);
+        final int messageClassId = buffer[3] & 0xFF;
+        this.messageClass = SerialMessageClass.getMessageClass(messageClassId);
```

Comment:
Ok, but doesn’t this then result in a “Serial message class not implemented” WARNing since the message still can’t be processed. So I think you should now have two warnings in the log - one for the unknown label that you just added, and then the “not implemented” warning later on.  I think we should handle this in one place - we could catch this exception in the command processor and log the error there rather than logging the error here, then failing further in the processing anyway?